src/universal/components/Masthead/DefaultMasthead/DefaultMasthead.mobile.tsx

```diff
@@ -37,7 +49,11 @@ const DefaultMasthead = ({
   searchRequested,
 }) => {
   return (
-    <MastheadBase>
+    <MastheadBase className=''>
+      {renderGlobalMenuModal({
```

Comment:
If you are trying to show/hide the component based on the state, I would suggest you go use `show` prop.
It's a small gimmick and intuitive from my humble opinion.

```jsx
<Component show={booleanValue} />

const Component = ({ booleanValue }) => !!booleanValue && <Foo />
```